homeassistant/components/ipma/__init__.py

```diff
@@ -65,3 +67,22 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data.pop(DOMAIN)
 
     return unload_ok
+
+
+class IPMADevice(Entity):
```

Comment:
It is our standard to move base entities in a separate module (`entity.py`). We should move this into its own module.